src/components/inherited-map/models/map-store.ts

```diff
@@ -181,7 +181,7 @@ export const MapStore = types
       geometry: {
         type: "Circle",
         radius:
-          calculateMetersPerPixelInWgs84(acc.point.latitude, self.zoom) *
+          calculateMetersPerPixelInWgs84(acc.point.latitude, zoom) *
```

Comment:
Takes `zoom` from fn param, which is the same as `self.zoom`, but more transparent